downstream/snippets/deprecated-features.adoc

```diff
@@ -0,0 +1 @@
+Deprecated functionality is still included in {PlatformNameShort} and continues to be supported. However, it will be removed in a future release of this product and is not recommended for new deployments. 
```

Comment:
Suggestions for clarity:

- 'it' -> 'the functionality'
- 'this product -> platform name or 'the platform'